ocaml/xenopsd/c_stubs/xenctrlext_stubs.c

```diff
@@ -672,16 +672,16 @@ CAMLprim value stub_xenforeignmemory_unmap(value fmem, value mapping)
 }
 
 CAMLprim value stub_xenctrlext_domain_claim_pages(value xch_val, value domid_val,
-        value nr_pages_val)
+        value numa_node_val, value nr_pages_val)
 {
-        CAMLparam3(xch_val, domid_val, nr_pages_val);
+        CAMLparam4(xch_val, domid_val, numa_node_val, nr_pages_val);
         int retval, the_errno;
         xc_interface* xch = xch_of_val(xch_val);
         uint32_t domid = Int_val(domid_val);
         unsigned long nr_pages = Long_val(nr_pages_val);
 
         caml_release_runtime_system();
-        retval = xc_domain_claim_pages(xch, domid, nr_pages);
+        retval = xc_domain_claim_pages(xch, domid, numa_node, nr_pages);
```

Comment:
I think we should use `Nativeint_val` on this `value` before releasing the lock?

Reply:
Looks like some bad patch squashing has happened.

Reply:
yeah, need to fix it!

Reply:
Thanks, I've fixed the commit